agent/handlers/gql/schema_test.go

```diff
@@ -222,11 +308,32 @@ var (
 	expectedFailedPulledContainerResponse = gqlResponse{
 		Errors: []string{"Unable to get container name mapping for task" + taskARN},
 	}
-	containerQueryGQL         = "{Container{DockerId,Name,DockerName,KnownStatus,DesiredStatus,CreatedAt,StartedAt,FinishedAt,ExitCode,Image,ImageID,Labels,Type,LogDriver,LogOptions,Limits,ContainerARN,Networks}}"
-	taskQueryGQL              = "{Task{Cluster,TaskARN,Family,Revision,DesiredStatus,KnownStatus,Limits,PullStartedAt,PullStoppedAt,ExecutionStoppedAt,AvailabilityZone,LaunchType,Containers{DockerId,Name,DockerName,KnownStatus,DesiredStatus,CreatedAt,StartedAt,FinishedAt,ExitCode,Image,ImageID,Labels,Type,LogDriver,LogOptions,Limits,ContainerARN,Networks}}}"
-	containerStatsQueryGQL    = "{Container{Stats}}"
-	containerNetworksQueryGQL = "{Container{Networks}}"
-	tagsQueryGQL              = "{Task{ContainerInstanceTags,TaskTags}}"
+	expectedFailedContainerByIdResponse1 = gqlResponse{
+		Errors: []string{"Unable to find container: fakeId"},
+	}
+	expectedFailedContainerByIdResponse2 = gqlResponse{
+		Errors: []string{"Container query only allows for at most one argument"},
+	}
+	expectedFailedContainerByIdResponse3 = gqlResponse{
+		Errors: []string{"Unable to find container: fakeName"},
+	}
+	containerQueryGQL            = "{Container{DockerId,Name,DockerName,KnownStatus,DesiredStatus,CreatedAt,StartedAt,FinishedAt,ExitCode,Image,ImageID,Labels,Type,LogDriver,LogOptions,Limits,ContainerARN,Networks,Ports,Volumes,Health}}"
```

Comment:
ca we format these queries with multiline strings (i.e. strings delimited with backticks `)